### DISCUSSION TITLE
Lets Talk Future!

### DISCUSSION THREAD

**Discussion by CSBrad** [CSBrad/ASAM#2] [Ideas]
Let me know where you want ASAM to go, please give me feature ideas. just things you would like to see added!

**Comment by SanR-Bylbylyator** [comment]
I would like to see localization in Russian. Or even let users add their own language presets by translating the English version. It would be great to add a mod manager and a system for checking input data into the game.ini and gameusersettings.ini files.

**Reply by John-T-Angle** [reply]
that option is already available for some languages

**Comment by Sydchilled** [comment]
A version for linux would be very cool,

**Reply by CSBrad** [reply]
Something in the works for that, will put it announcements closer to the time :)
